docs/concepts/time-and-windows-in-ksqldb-queries.md

```diff
@@ -147,8 +147,6 @@ rules.
   records, the timestamp of the output record is assigned `max(left.ts, right.ts)`.
 * For stream-table joins, the output record is assigned the timestamp from the
   stream record.
-* For aggregations, the timestamp of the resulting update record is
-  taken from the latest input record that triggered the update.
```

Comment:
Side cleanup. We have two bullet points for aggregation and the second one below is correct.